### ISSUE TITLE
Video card  is currently rendering two images without alt text -- Can't access player with screen readers

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wai-easy-checks#231]
Refs:

- Code: https://github.com/w3c/wai-easy-checks/blob/c36f280a35a1fceb5d1c901f4eb945f3737859f2/content/index.md?plain=1#L100
- [Preview](https://deploy-preview-131--wai-easychecks.netlify.app/test-evaluate/easy-checks/)

This code is currently rendering two images without alt text. Presumably one for the video thumbnail and another one for the play button.

```
<div class="video-card">
      <img class="video" src="https://i.ytimg.com/vi_webp/IZp8eUAu450/maxresdefault.webp">
      <img class="play-button" src="https://www.starpng.com/public/uploads/preview/red-play-button-png-101576847279kuxmycuxzj.png">
  </div>
```

I cannot access the player buttons with my screen reader.

I am not sure where this is coming from though. Is this the expected behavior?

**Comment by iadawn** [comment]
Video card was previously just a placeholder. This has now been replaced with the correct embedded video code
